content/docs/guides/10-minute-tutorial.md

```diff
@@ -442,11 +442,16 @@ You should see something like the following:
  T E S T S
 -------------------------------------------------------
 
-Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
+Scenario: The example                       # hellocucumber/example.feature:3
```

Comment:
This is incorrect because this means you've copied the stuff over. To start with you **should** have an empty project

Reply:
It does look like the tutorial is somewhat out of date though. It still references JUnit 4.